scripts/platform_defs_gen.c

```diff
@@ -14,6 +14,20 @@ int main() {
 
     printf("#define PLAT_CPU_NUM (%ld)\n", platform.cpu_num);
     printf("#define PLAT_BASE_ADDR (0x%lx)\n", platform.regions[0].base);
+
+    for(size_t i = 1; i < platform.region_num; i++)
+    {
+        /*
+         * Selects the first memory region with RWX (read, write, execute) permissions, and defines
+         * it as PLAT_DATA_MEM. This region is considered the main data memory that Bao will use
+         * for its own purposes.
+         */
+        if(platform.regions[i].perms == RWX)
+        {
+            printf("#define PLAT_DATA_MEM (0x%lx)\n", platform.regions[i].base);
+            break;
+        }
+    }
```

Comment:
I am thinking we could also allow a user to define the the data base in the hyp section of the configuration. Otherwise it is too rigid to make the user use that region for hypervisor data in all configurations

Reply:
Fixed in 187e1ca9